spec/models/calculation_spec.rb

```diff
@@ -17,4 +17,31 @@
       .t("#{LOCAL_PREFIX_CALCULATION}.value.too_short"))
     }
   end
+
+  describe '#result' do
+    it 'Should be valid' do
```

Comment:
Here and below https://github.com/rubocop/rspec-style-guide#should-in-example-docstrings